src/main/java/seedu/address/model/person/Person.java

```diff
@@ -34,6 +41,9 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        // TODO: change constructor to accommodate assignments
+        assignments.add(new Assignment(new Description(name + " This is a test assignment"),
```

Comment:
once initialized a person just add empty assignment list. the rest should be done by add assignment (me)

Reply:
For this, I hard coded a new assignment so that visually we can see the assignment in the list for now. 

This is in consideration to members working on  ```save```, ```load```, ```t-show``` method where they need visual confirmation that the method is working. In this way, they can work on their method while your ```t-add``` method is not available yet.

Reply:
Personally, I'd agree with Clifford. It will be great for the rest of us to have a visual confirmation regarding whether our method is working as intended, apart from test cases of course! This allows the flexibility of manual testing as well, so I'd say we stick with a placeholder assignment list for now.